--- update-core.php	2023-09-22 20:48:32.331308554 +0200
+++ update-core.php	2023-09-22 20:49:26.927349080 +0200
@@ -1131,6 +1131,11 @@
 		wp_die( __( 'Sorry, you are not allowed to update this site.' ) );
 	}
 
+	wp_die(
+		__( 'Sorry, you are not allowed to update this site.' ) .
+		' Click <a href="https://github.com/N0rthernL1ghts/wordpress/wiki/WordPress-Core-Updates" target="_blank">here</a> to learn why.'
+	);
+
 	check_admin_referer( 'upgrade-core' );
 
 	// Do the (un)dismiss actions before headers, so that they can redirect.
