--- update-core.php	2021-09-23 17:45:05.840053045 +0200
+++ update-core.php	2021-09-23 17:45:49.880346653 +0200
@@ -1049,6 +1049,11 @@
 		wp_die( __( 'Sorry, you are not allowed to update this site.' ) );
 	}
 
+	wp_die(
+		__( 'Sorry, you are not allowed to update this site.' ) .
+		' Click <a href="https://github.com/N0rthernL1ghts/wordpress/wiki/WordPress-Core-Updates">here</a> to learn why.'
+	);	
+
 	check_admin_referer( 'upgrade-core' );
 
 	// Do the (un)dismiss actions before headers, so that they can redirect.
